packages/reporting-server/rest_server/test_orm_config.py

```diff
@@ -0,0 +1,12 @@
+# conflicts with isort because of local non-relative import
+# pylint: disable=wrong-import-order
```

Comment:
There doesn't seem to be any imports here

Reply:
Sorry this is a leftover